kirin/new_relic.py

```diff
@@ -50,6 +54,34 @@ def init(config_file):
         logger.warning("newrelic hasn't been initialized")
 
 
+def must_never_log(exception):
+    # Not tracking requests to wrong endpoints
+    if isinstance(exception, NotFound):
+        return True
+    # Not tracking requests with wrong HTTP verbs
+    if isinstance(exception, MethodNotAllowed):
+        return True
+    # Any other exception should be logged
+    return False
+
+
+def is_invalid_input_exception(exception):
+    if isinstance(exception, InvalidArguments):
+        return True
+    # Any other exception is not related to an invalid input
+    return False
+
+
+def must_log_apm(exception):
```

Comment:
This function will let us know if we have to record an event if it isn't among the exception we aren't interested in so I'll suggest
```suggestion
def allow_apm_logging(exception):
```